src/main/java/net/minecraftforge/client/event/ClientChatEvent.java

```diff
@@ -20,13 +20,15 @@
 package net.minecraftforge.client.event;
 
 import com.google.common.base.Strings;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.ForgeEventFactory;
 import net.minecraftforge.eventbus.api.Cancelable;
 import net.minecraftforge.eventbus.api.Event;
 
 /**
  * ClientChatEvent is fired whenever the client is about to send a chat message or command to the server. <br>
  * This event is fired via {@link ForgeEventFactory#onClientSendMessage(String)},
- * which is executed by {@link GuiScreen#sendChatMessage(String, boolean)}<br>
+ * which is executed by {@link net.minecraft.client.gui.screens.Screen#sendMessage(String, boolean)}<br>
```

Comment:
Needs to be consistency whether you should import to link or not.

Reply:
What would be your recommendation? I'm of the opinion to use the imports myself, partly due to preference and keeping the javadocs small.

Reply:
Went with using imports.